### PR TITLE
refactor: welcome post card to support freeform posts

### PR DESCRIPTION
**Pull request by sshanzel**
## Changes
- Configured line clamping to be up until 9 as required by our Card when there is no title.
- Fixed the Welcome Post modal to handle null cover image.

Reference: https://www.sketch.com/s/2867ec63-0966-4647-b1d7-b6fe696ba251/a/qbbzMDV#Inspect

Previews:

Screenshot 2023-05-30 at 11 41 04 AM

Screenshot 2023-05-30 at 11 36 29 AM

### Describe what this PR does
- Short and concise, bullet points can help
- Screenshots if applicable can also help

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-1336 #done

### CODE REVIEW

File: packages/shared/src/graphql/feed.ts [hunk 1]
```diff
@@ -178,7 +178,7 @@ export const SOURCE_FEED_QUERY = gql`
       ...FeedPostConnection
     }
   }
-  ${getFeedPostFragment('pinnedAt')}
+  ${getFeedPostFragment('pinnedAt content')}
```
Comment (capJavert) [inline packages/shared/src/graphql/feed.ts:181]: Not specific to this PR but I was thinking by returning `content` for feed listing which has `4000` char limit we might have considerably bigger responses if there is more freeform posts in the future. @idoshamun FYI interested what you think.

**Review by capJavert — APPROVED** [review]
Looking good 🤘 can be merged I think but minor comment for future proofing.

File: packages/shared/src/graphql/feed.ts (hunk 1 above)
Comment (idoshamun) [reply packages/shared/src/graphql/feed.ts:181]: That's a good point! I also think that we don't need for post modal in 99% of the cases. We will need it only for edit mode. Let's log the post modal as something to fix and in this PR let's remove the content for the feed

Comment (sshanzel) [reply packages/shared/src/graphql/feed.ts:181]: For the feed, we will need the content (switch to using `contentHtml` now) on the feed to display when there is no cover image on the post. Do we maybe want to make it conditional as to only return it when there is no image?

**Pull request merged**